<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mutual Funds Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>85</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Morning Stars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRating1">
       <property name="text">
        <string>★</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRating2">
       <property name="text">
        <string>★★</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRating3">
       <property name="text">
        <string>★★★</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRating4">
       <property name="text">
        <string>★★★★</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRating5">
       <property name="text">
        <string>★★★★★</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="msRatingNA">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>711</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sliderLayout">
     <item>
      <widget class="QCheckBox" name="bondsStocksCheck">
       <property name="text">
        <string>Bonds &lt;-&gt; Stocks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="minSBRatio">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="maxSBRatio">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>911</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Exchange Market Countries</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="countriesContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>448</width>
            <height>101</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="selAllCountries">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="unSelAllCountries">
           <property name="text">
            <string>Unselect All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showAllCountries">
           <property name="text">
            <string>Show All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Exchange Markets</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="marketContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>447</width>
            <height>101</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="selAllMarkets">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="unSelAllMarkets">
           <property name="text">
            <string>Unselect All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showAllMarkets">
           <property name="text">
            <string>Show All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="quoteCount">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>370</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QCheckBox" name="doSaveCSV">
       <property name="text">
        <string>Save CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeData">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>FIND</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>291</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Stock Style</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Capitalization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="capCombo">
           <item>
            <property name="text">
             <string>N/A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Small</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Large</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="styleCombo">
           <item>
            <property name="text">
             <string>N/A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Value</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blend</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Growth</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Bond Style</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Credit Quality</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="qualityCombo">
           <item>
            <property name="text">
             <string>N/A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Low</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medium</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Interest Rate Sensitivity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sensitivityCombo">
           <item>
            <property name="text">
             <string>N/A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Limited</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Moderate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Extensive</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>241</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QCheckBox" name="minYieldCheck">
         <property name="text">
          <string>Yield &gt;=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="minYield">
         <property name="maximum">
          <double>99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QCheckBox" name="maxExpenseCheck">
         <property name="text">
          <string>Expense Ratio &lt;=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxExpense">
         <property name="maximum">
          <double>99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QCheckBox" name="maxInvestCheck">
         <property name="text">
          <string>Min Invest &lt;=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxInvest">
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fund Types</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="fundTypeContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>237</width>
          <height>103</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="selAllFundType">
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unSelAllFundType">
         <property name="text">
          <string>Unselect All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
